enroll/service.go

```diff
@@ -145,3 +148,57 @@ func (svc service) Enroll(ctx context.Context) (Profile, error) {
 
 	return *profile, nil
 }
+
+func (svc service) OTAEnroll(ctx context.Context) (Payload, error) {
+	payload := NewPayload("Profile Service")
+	payload.PayloadIdentifier = "com.github.micromdm.ota.profile-service"
+	payload.PayloadDisplayName = "MicroMDM Profile Service"
+	payload.PayloadDescription = "Profile Service enrollment"
+	payload.PayloadOrganization = "MicroMDM"
+	payload.PayloadContent = ProfileServicePayload{
+		URL:              svc.URL + "/ota/phase23",
+		Challenge:        "",
+		DeviceAttributes: DefaultProfileServiceDeviceAttributes,
+	}
+
+	// yes, this is a bare Payload, not a Profile
+	return *payload, nil
+}
+
+func (svc service) OTAPhase2(ctx context.Context) (Profile, error) {
+	profile := NewProfile()
+	profile.PayloadIdentifier = "com.github.micromdm.micromdm.ota-scep"
+	profile.PayloadOrganization = "MicroMDM"
+	profile.PayloadDisplayName = "Enrollment Profile"
+	profile.PayloadDescription = "The server may alter your settings"
+	profile.PayloadScope = "System"
+
+	scepContent := SCEPPayloadContent{
+		URL:      svc.SCEPURL,
+		Keysize:  1024,
```

Comment:
I'd love to. I merely copied it from the "normal" enrollment profile a few lines above so figured there was a reason it was used. As well Apple has this to say on it [here](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/iPhoneOTAConfiguration/profile-service/profile-service.html#//apple_ref/doc/uid/TP40009505-CH2-SW35):

> Key size requests the device to generate a keypair of a certain size. Only 1024-bit and 2048-bit key sizes should be used. Keys larger than 2048 bits are not supported. In general, 1024-bit keys are recommended because of the overhead involved in generating 2048-bit keys.